7.2.0/postgresql/delete_um_tenant.sh

```
@@ -0,0 +1,88 @@
+  else
+    FAILED_LOG="$FAILED_LOG_DIR/failed_tenant_${TENANT_ID}.log"
+    echo "$DELETE_PROCEDURE" > "$FAILED_LOG"
+    echo "Failed to delete UM_TENANT entry for TENANT_ID=$TENANT_ID. Check log: $FAILED_LOG" | tee -a "$SUMMARY_FILE"
+  fi
+done
+
+# Unset PGPASSWORD to avoid leaving it in the environment
+unset PGPASSWORD
+
+echo "UM_TENANT deletion process completed. Summary available at $SUMMARY_FILE."
+
```

Shellcheck (0.11.0)

[error]  Fix any mentioned problems and try again.

(SC1072)